### PR TITLE
Few simplifications

### PR DESCRIPTION
**Pull request by jotak**
This is a bunch of little things:

- ParsedStream model now has 'ipfix' and 'timestamp' at its root instead of nested under 'value'. Also, low-cased 'ipfix' as per json conventions. Added a 'key' field that should identify the flow.
- In react table, use that flow key field for row identification (using the index could be buggy on refresh / sorting etc. as this index may change over time for a given flow)
- Remove disabled no-explicit-any when possible
- Created "netflow-table" directory
- Add lambdas to Columns to avoid big switch block
- Add lodash types definitions
- Removed BrowserHistory prototype patch, as it is apparently useless for us

### CODE REVIEW

File: web/src/api/loki.ts
Comment (jotak) [inline web/src/api/loki.ts:26]: Do you think we should rename more stuff here? I realize `ParsedStream` was probably a poor choice ...
What about `IPFIXRecord` instead? Maybe also rename `IPFIXStream` to `IPFIXBlob` or something similar?

File: web/src/components/__tests-data__/columns.tsx
Comment (jpinsonneau) [inline web/src/components/__tests-data__/columns.tsx:5]: In that case I would set default value with `(k: string) => k` and keep argument only in `NetflowTraffic` component :
`export const getDefaultColumns = (t: TFunction = (k: string) => k): Column[] => {`

File: web/src/components/filters-toolbar.tsx
Comment (jpinsonneau) [inline web/src/components/filters-toolbar.tsx:387]: :+1:

File: web/src/utils/columns.ts
Comment (jpinsonneau) [inline web/src/utils/columns.ts:185]: Should we move this to another file in `utils` folder ? 

We already have separate files for `ip` and `port`. Maybe we should make a `filter.ts` file and put everything inside ?

Comment (jotak) [reply web/src/utils/columns.ts:185]: good point i'll do that

File: web/src/components/__tests-data__/columns.tsx
Comment (jotak) [reply web/src/components/__tests-data__/columns.tsx:5]: hmm i'm not sure, because it isn't meant to be unset in the nominal case. It's more here in tests that is a special case where we override that i18n TFunction, but in any other usage, it must be provided

Comment (jpinsonneau) [reply web/src/components/__tests-data__/columns.tsx:5]: ok that makes sense

File: web/src/utils/columns.ts
Comment (jotak) [reply web/src/utils/columns.ts:185]: after looking at that, `port.ts` has more than just the comparison function, so I'd rather keep it and organize per data type (create `protocol.ts`)

Comment (jpinsonneau) [reply web/src/utils/columns.ts:185]: sure, thx

**Comment by mariomac** [comment]
Overall looks good to me. Let's wait for @jpinsonneau 's ok for final approval

**Review by jpinsonneau — APPROVED** [review]
That's good for me too, thanks for the changes :+1:

**Comment by jotak** [comment]
Thanks!
/approve

**Pull request merged**
